RELEASE-NOTES.md

```diff
@@ -1,3 +1,32 @@
+### 0.11.0
+#### Compatibility
+- golem-messages: 2.24.0
+- golem-smart-contracts-interface: 1.6.1
+
+#### Endpoints and uses cases in general
+- Bugfix: Change concent response from ServiceRefused to Http404 (#1051).
+- Bugfix: Validate protocol version (#1057).
+- Bugfix: Fix non nesting atomic to verify transaction nesting in runtime instead of on startup (#1053).
+- Bugfix: Wrong jpeg handling in verification use case (#1046).
+
+#### Payments
+- Preventing multiple settlement payments for the same subtasks (#1009, #1039).
```

Comment:
Those are two completely separate things. They were done in one pull request but in the changelog they should not be bunched together:
- #1009 was a revert of #982 which never appeared in any tagged release. So these two shouldn't even be on this list. And they should have no milestone assigned.
- The description is fine for #1039.

Reply:
Done